### PR TITLE
UL&S iCloud Keychain: implements 2FA support.

### PR DESCRIPTION
**Pull request by diegoreymendez**
Ref: https://github.com/wordpress-mobile/WordPress-iOS/issues/14734
WPiOS PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/14783

This PR implements 2FA support for iCloud Keychain.

Please refer to the WPiOS PR for testing instructions.

### CODE REVIEW

File: WordPressAuthenticator/Unified Auth/StoredCredentialsAuthenticator.swift [hunk 1]
```diff
@@ -120,6 +124,18 @@ class StoredCredentialsAuthenticator: NSObject {
 @available(iOS 13, *)
 extension StoredCredentialsAuthenticator: LoginFacadeDelegate {
     func needsMultifactorCode() {
+        guard let loginFields = loginFields else {
+            return
+        }
+        
+        guard let vc = Login2FAViewController.instantiate(from: .login) else {
```
Comment (mindgraffiti) [inline WordPressAuthenticator/Unified Auth/StoredCredentialsAuthenticator.swift]: This should use the new UI. The new VC is called TwoFAViewController.

**Review by mindgraffiti — CHANGES_REQUESTED** [review]
This is using the old 2fa UI. Since this is a completely new feature I expect it to use the new UI.

File: WordPressAuthenticator/Unified Auth/StoredCredentialsAuthenticator.swift (hunk 1 above)
Comment (diegoreymendez) [reply WordPressAuthenticator/Unified Auth/StoredCredentialsAuthenticator.swift]: Good observation.  I didn't realize this was the old VC.

I also updated the expected tracked events in the WPiOS PR description.

**Comment by diegoreymendez** [comment]
Ready for another look.

**Review by mindgraffiti — APPROVED** [review]
✅  Code
✅  UX & UI 
✅  Tracks Events

**Pull request merged**